src/resource.js

```diff
@@ -46,15 +45,14 @@ export default class Resource {
    *
    * @returns {String}
    */
-  get type() {
+  get typeOfResourcePath() {
```

Comment:
Please follow https://github.com/frictionlessdata/stack/issues/2 (`Resource.type` for now)